tamr_client/categorization/project.py

```diff
@@ -21,6 +25,39 @@ def _from_json(url: URL, data: JsonDict) -> CategorizationProject:
     )
 
 
+def create(
+    session: Session,
+    instance: Instance,
+    name: str,
+    description: Optional[str] = None,
+    external_id: Optional[str] = None,
+    unified_dataset_name: Optional[str] = None,
+) -> Project:
+    """Create a Categorization project in Tamr
+    Args:
+        instance: Tamr instance
+        name: Project name
+        description: Project description
+        external_id: External ID of the project
+        unified_dataset_name: Unified dataset name. If None, will be set to project name + _'unified_dataset'
+
+    Returns:
+        A Categorization project object
+
+    Raises:
```

Comment:
Missing exception type (I think there are 2 other occurrences of this in the code)

Reply:
removed the exception type from the docstring of this function since the actual exception is raised by the _create() method

Reply:
The issue is that if users look at the generated reference docs, the exception will be missing, so we need to include for all user-facing functions.

One thought: the docstrings seem repetitive, so we could write the docstring as a plain string once and use it for multiple functions via the `__doc__` property. See https://softwareengineering.stackexchange.com/a/327664/94802 . Not sure if I like this approach or not yet, but its an option.

Reply:
I think i prefer being rather verbose than referring the user to docstrings of other functions to figure out what errors could be thrown etc.